### ISSUE TITLE
How to use AdamW properly?

### ISSUE THREAD

**Issue by yuvaramsingh94** [tensorflow/addons#1949]
Hi all,
   i have a common question. while using AdamW optimizer from tfa, should i set ``kernel_regularizer`` to something specific in Conv2D layer or it can automatically calculate the weight decay. also i am using pre-trained CNN models. this information will let me know weather i should set it or use the naive pre-trained model.  

thanks
yuvaram

**Comment by seanpmorgan** [comment]
Hi, 

Closing as this is a duplicate of https://github.com/tensorflow/addons/issues/844
